cert-expiry-finder

```diff
@@ -294,10 +294,29 @@ expiry_after() {
     esac
 }
 
+list_k8s_cert_expiry_days() {
+    local name namespace notafter unixtime delta context
+    if ! which kubectl; then
+        return
+    fi
+    kubectl config get-contexts --no-headers -oname | while read context
```

Comment:
Generally `read -r` unless you know you don't want `-r`:
```
              -r     Backslash does not act as an escape character.  The backslash is considered to
                     be  part of the line.  In particular, a backslash-newline pair may not then be
                     used as a line continuation.
```

Reply:
changed